docs/accounts/account-secrets/export_view_only_account_package.md

```diff
@@ -31,38 +31,12 @@
             "method": "import_view_only_account",
             "params": {
                 "account": {
-                    "object": "view_only_account",
-                    "account_id": "6d95067c5fcc0dd7bbcdd42d49cc3571fe1bb2597a9c397c75b7280eca534208",
```

Comment:
Are these changes relevant to the multi-account support? Can we pull out other changes to another PR?